pages/index.js

```diff
@@ -1,63 +1,98 @@
 import styled from "styled-components";
-import React from "react";
 import { useState } from "react";
+import Link from "next/link";
 
-export default function Landingpage() {
-  const [output, setOutput] = useState("");
-  const handleSubmit = (event) => {
-    event.preventDefault();
-    const entry = event.target.elements.text.value;
-    setOutput(entry);
-  };
-  return (
-    <>
-      <container>
-        <Title>Your Travel Journal</Title>
-        <Form
-          onSubmit={handleSubmit}
-          aria-labelledby="Where is your journey going?"
-        >
-          <Label htmlFor="text">City or Country?</Label>
-          <input
-            type="text"
-            required
-            id="text"
-            name="text"
-            placeholder="enter your destination..."
-          />
-          <Button type="submit">Submit</Button>
-          <p>{output}</p>
-        </Form>
-      </container>
-    </>
-  );
-}
-
-const Title = styled.h1`
-  font-size: 1.5em;
+const Header = styled.header`
+  font-family: monda;
+  font-size: 2em;
+  color: #e7e4d8;
+  font-weight: bold;
   text-align: center;
+  background-color: #78c9cc;
 `;
 
 const Form = styled.form`
+  color: black;
   display: grid;
   width: 70%;
   margin: 5px auto;
   padding: 10px;
-  border: 1px solid rgb(37, 38, 41);
   border-radius: 5px;
   text-align: center;
 `;
 
 const Label = styled.label`
   display: center;
-  padding: 10px;
-  border-radius: 4px;
-  font-size: 1em;
+  padding: 20px;
+  font-size: 16pt;
```

Comment:
Is 16pt valid, or do you mean 16px? I never seen pt and if it works it's fine .

Reply:
yes, it works, its a size unit..

Reply:
Ok I lerned somethin new... never seen that before.